ci/ci_cli.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+
+from concurrent import futures
+
+MAX_WORKERS = 10
+
+JOB_TYPE_INSTALL = 'install'
+JOB_TYPE_DIAGNOSTIC = 'diagnostic'
+
+JOB_TYPES = [JOB_TYPE_INSTALL, JOB_TYPE_DIAGNOSTIC]
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def install(service_name):
+    import sdk_install
+
+    log.info("Starting %s installation.", service_name)
+
+    sdk_install.install(
+        service_name,
+        service_name,
+        0,
+        package_version=sdk_install.PackageVersion.LATEST_UNIVERSE
+    )
+
+
+def diagnostic(service_name):
+    log.info("Starting %s diagnostic.", service_name)
+
+    src = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        'python', 'create_service_diagnostics_bundle.sh'
+    )
+
+    cmd = [
+        src,
+        "--package-name={service}".format(service=service_name),
+        "--service-name={service}".format(service=service_name),
+        "--yes"]
+
+    subprocess.Popen(cmd).wait()
```

Comment:
This swallows errors. You need:
```suggestion
    subprocess.check_call(cmd)
```
instead.

Reply:
changed, thanks for the suggestion